app/views/classified_listings/index.html.erb

```diff
@@ -21,18 +21,22 @@
 <% end %>
 
 <div class="home">
-  <div class="classifieds-container" id="classifieds-index-container"
-    data-category="<%= params[:category] %>" data-listings="<%= @classified_listings.to_json(
-            only: %i[title processed_html tag_list category id user_id],
-            include: {
-              author: { only: %i[username name], methods: %i[username profile_image_90] },
-            },
-          ) %>"
-    data-allcategories="<%= ClassifiedListing.categories_for_display.to_json %>"
-  >
+  <div class="classifieds-container"
+       id="classifieds-index-container"
+       data-category="<%= params[:category] %>" data-listings="<%= @classified_listings.to_json(
+        only: %i[title processed_html tag_list category id user_id slug],
+        include: {
+          author: { only: %i[username name], methods: %i[username profile_image_90] }
+        },
+      ) %>"
+    data-allcategories="<%= ClassifiedListing.categories_for_display.to_json %>">
     <div class="classified-filters">
     <div class="classified-filters-categories">
-      <a href="/listings" class="<%= "selected" if params[:category].blank? %> data-no-instant="true">all</a><% ClassifiedListing.categories_for_display.each do |cat| %><a href="/listings/<%= cat[:slug] %>" class="<%= "selected" if params[:category] == cat[:slug] %> data-no-instant="true"><%= cat[:name] %></a><% end %><a href='/listings/new' class='classified-create-link'>Create a Listing</a>
+      <a href="/listings" class="<%= "selected" if params[:category].blank? %>" data-no-instant="true">all</a>
```

Comment:
This refactor is causing it to "jump". The whitespace is treated differently in the JS vs here so I'm not sure how to make it work with this refactoring. But either way, needs to be sure it won't jump on load.